ament_copyright/package.xml

```diff
@@ -5,15 +5,12 @@
   <description>The ability to check every source file contains copyright reference in the ament buildsystem.</description>
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
-  <license>BSD</license>
 
-  <buildtool_depend>ament_cmake_core</buildtool_depend>
-  <buildtool_depend>ament_cmake_python</buildtool_depend>
-  <buildtool_depend>ament_cmake_test</buildtool_depend>
+  <buildtool_depend>python3</buildtool_depend>
 
-  <buildtool_export_depend>ament_cmake_test</buildtool_export_depend>
+  <buildtool_export_depend>python3</buildtool_export_depend>
```

Comment:
Does it make sense to have `python3` be a buildtool dependency? If the intention is to be able to run this python script after building you'll need at least a `run_depend` on a target platform specific version of python.